### ISSUE TITLE
Shiny fails when there's a .Rmd file in Home directory

### ISSUE THREAD

**Issue by yuvipanda** [utoronto-2i2c/jupyterhub-deploy#64]
If I have *any* `.Rmd` files in $HOME, that seems to be loaded by shiny first, rather than the path it's actually supposed to start in. This causes weird issues where things fail for *some* people, but not all.

**Comment by ntaback** [comment]
Yup, that's exactly what happened to me.  One problem is that when a user creates a new .Rmd file in RStudio the default is to save it to $HOME.
